chain/x/mint/alias.go

```diff
@@ -44,8 +46,9 @@ var (
 )
 
 type (
-	Keeper       = keeper.Keeper
-	GenesisState = types.GenesisState
-	Minter       = types.Minter
-	Params       = types.Params
+	Keeper            = keeper.Keeper
+	GenesisState      = types.GenesisState
+	Minter            = types.Minter
+	Params            = types.Params
+	MsgMintCoinsToAcc = types.MsgMintCoinsToAcc
```

Comment:
It is not about minting coins to amount, so better use "MsgWithdrawCoinsToAccFromTreasury" or smth similar